src/mlpack/methods/linear_svm/linear_svm.hpp

```diff
@@ -196,19 +199,22 @@ class LinearSVM
   /**
    * Train the Linear SVM with the given training data.
    *
-   * @tparam OptimizerType Desired optimizer
-   * @param data Input training features. Each column associate with one sample
+   * @tparam OptimizerType Desired optimizer.
+   * @tparam CallbackTypes Types of Callback Functions.
+   * @param data Input training features. Each column associate with one sample.
    * @param labels Labels associated with the feature data.
    * @param numClasses Number of classes for classification.
    * @param lambda L2-regularization constant.
    * @param optimizer Desired optimizer.
+   * @param callbacks Callback Functions.
```

Comment:
In another PR, @rcurtin suggested to add a reference to the ensmallen documentation: https://github.com/mlpack/mlpack/pull/2074#discussion_r341866123. I think we should do the same here as well.